src/test/test-models/good-standalone-functions/integrate.stanfuncs

```diff
@@ -1,5 +1,5 @@
 functions {
-
+/*
```

Comment:
This was commented out as standalone functions tests are currently run with stanc2 and stanc2 will no longer support integrate_ functions (due to changes for the variadic ODE).

Once stanc3 will support standalone functions this will be uncommented back.

Reply:
This can be uncommented